### PR TITLE
feat: automate greeting using github action

### PR DESCRIPTION
**Pull request by dakshsinghrathore**
This PR fixes the issue #8  , kindly review it and merge.

You need to give `read and write permission` to the bot to make comments hence kindly go to the repo  `settings/actions/General`  and under  `Workflow Permissions`  grant  `Read and write` permissions and then `save`.

like this ⬇️

image

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Congratulations, @${{ github.actor }}! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!
+          pr-message: |
+            Great job, @${{ github.actor }}! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.
```
Comment (Doraemon012) [inline .github/workflows/greetings.yml:23]: > Great job, @${{ github.actor }}! 🎉 Thank you for submitting your pull request

Make it 
**Great job, @${{ github.actor }}! 🎉 Thank you for submitting a pull request**

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Congratulations, @${{ github.actor }}! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!
```
Comment (Doraemon012) [inline .github/workflows/greetings.yml:21]: > Congratulations, @${{ github.actor }}! 🎉 Thank you for creating your issue.

Make it
**Congratulations, @${{ github.actor }}! 🎉 Thank you for creating an issue.**

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Kindly make the requested changes.

File: .github/workflows/greetings.yml (hunk 2 above)
Comment (dakshsinghrathore) [reply .github/workflows/greetings.yml:21]: Will make the changes by tonight !

**Review by Doraemon012 — APPROVED** [review]

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the required changes.

**Comment by Doraemon012** [comment]
@dakshsinghrathore  Please update your branch, it has been outdated.

This happens when you have pushed commits to your branch and the remote repo has also been updated. That's why, it shows something like `your branch is 1 commit behind and 3 commits ahead of...`

To resolve this, go to your branch, then click on `Sync Fork` and then click on `Update branch`.

You can use the below screen shot and recording as reference. Here, the branch `patch-1` is one commit ahead and 9 commits behind of the remote repository.

image

https://github.com/Tanay-ErrorCode/lupo-skill/assets/116624667/6fe638ca-bc95-4a5e-a43a-11d7757f21dd

**Comment by dakshsinghrathore** [comment]
@Tanay-ErrorCode I have fixed the required changes, also updated the main branch of my fork and then made the changes

**Pull request merged**